src/main/java/com/google/cloud/resourcemanager/ResourceManager.java

```diff
@@ -337,4 +338,20 @@ public static ProjectListOption fields(ProjectField... fields) {
    *     Platform Services</a>
    */
   List<Boolean> testPermissions(String projectId, List<String> permissions);
+
+  /**
+   * Returns the permissions and their results representing whether the caller has the permissions
+   * on the specified Organization.
+   *
+   * @param resource the organization's resource name, e.g. "organizations/123"
+   * @param permissions the set of permissions to check for the resource. Permissions with wildcards
+   *     (such as '*' or 'storage.*') are not allowed.
+   * @return the permissions and their results representing whether the caller has the permissions
+   *     on the specified Organization.
+   * @throws ResourceManagerException upon failure
+   * @see <a href=
+   *     "https://cloud.google.com/resource-manager/reference/rest/v1/organizations/testIamPermissions">
+   *     Resource Manager testIamPermissions</a>
+   */
+  Map<String, Boolean> testOrgPermissions(String resource, List<String> permissions);
```

Comment:
Not new in this PR, and shouldn't be fixed now, but the distinction between ResourceManager and ResourceManagerImpl is likely unnecessary and just makes the code more complicated for no particular reason.